src/DependencyInjection/Configuration.php

```diff
@@ -65,40 +66,58 @@ public function getConfigTreeBuilder() : TreeBuilder
      *
      * @since  2015-07
      *
-     * @return \Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition
-     * @return \Symfony\Component\Config\Definition\Builder\NodeDefinition
      * @throws \RuntimeException
+     *
+     * @return ArrayNodeDefinition
```

Comment:
Please use FQCN in annotations.

Reply:
Do you mean `\Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition` instead of `ArrayNodeDefinition`? As I know, it's no need to use full namespace, if class is imported. You can see many examples in symfony: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/DependencyInjection/ContainerBuilder.php#L448

Reply:
Do you mean \Symfony\Component\Config\Definition\Builder\ArrayNodeDefinition instead of ArrayNodeDefinition? As I know, it's no need to use full namespace, if class is imported. You can see many examples in symfony: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/DependencyInjection/ContainerBuilder.php#L448

Do you have any reasons to use full namespace instead of short one and importing in use section?

Reply:
It is only about readability. Every modern IDE understand use statements and in php it also works. But I would like to see always the FQCN directly without taking a deeper look into use statements. When you have a class name like "ArrayCollection" and this class name exists multiple times in your project (one in namespace/context "Items", one in "Users") than it is easier to see in which namespace/context this class is used.

Reply:
ok, agree with you. Fixed :slightly_smiling_face: